opm/grid/CpGrid.hpp

```diff
@@ -308,8 +308,12 @@ namespace Dune
         /// Create a cartesian grid.
         /// \param dims the number of cells in each cartesian direction.
         /// \param cellsize the size of each cell in each dimension.
+        /// \param shift Number of cell widths the left-most front-most lowest
+        ///             corner of the grid should be shifted to the right, back,
+        ///             and top
```

Comment:
I think I understood this, but maybe
"moves the origin of the grid in (i, j, k) coordinates relative to the default, so that if `shift == {si, sj, sk}` then the origin of the grid (i.e. the location of the corner with the minimum coordinates) is not (0.0, 0.0, 0.0) but (si * cellsize[0], sj * cellsize[1], sk * cellsize[2])"

Now if that is wrong, I truly did not understand the doc...

Reply:
I think we are getting there. Just not sure whether there is an "origin of a grid".

Reply:
thanks. I tried to improve the comment based on your input. seems much clearer now.